test/unit/rules/no-valueless-arguments-test.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const { ERROR_MESSAGE } = require('../../../lib/rules/no-valueless-arguments');
+const generateRuleTests = require('../../helpers/rule-test-harness');
+
+generateRuleTests({
+  name: 'no-valueless-arguments',
+
+  config: true,
+
+  good: [
+    '<SomeComponent @emptyString="" data-test-some-component />',
+    `<button type="submit" disabled {{on "click" this.submit}}></button>`,
+  ],
```

Comment:
Can we add some tests for curly components? `{{foo-component ... }}`

Reply:
@bmish I think it's good to land, for now we don't have AST logic (in this MR), related to curly stuff.